unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEmotesSectionController.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL.Emotes;
+using DCL.EmotesCustomization;
+using DCL.Tasks;
+using System;
+using System.Linq;
+using System.Threading;
+using UnityEngine;
+
+namespace DCL.Backpack
+{
+    public class BackpackEmotesSectionController : IBackpackEmotesSectionController
+    {
+        public event Action<string> OnNewEmoteAdded;
+        public event Action<string> OnEmotePreviewed;
+        public event Action<string> OnEmoteEquipped;
+        public event Action<string> OnEmoteUnequipped;
+
+        private DataStore dataStore;
+        private IUserProfileBridge userProfileBridge;
+        private IEmotesCatalogService emotesCatalogService;
+        private IEmotesCustomizationComponentController emotesCustomizationComponentController;
+        private CancellationTokenSource loadEmotesCts = new ();
+
+        public void Initialize(
+            DataStore dataStore,
+            Transform emotesSectionTransform,
+            IUserProfileBridge userProfileBridge,
+            IEmotesCatalogService emotesCatalogService)
+        {
+            this.dataStore = dataStore;
+            this.userProfileBridge = userProfileBridge;
+            this.emotesCatalogService = emotesCatalogService;
+
+            emotesCustomizationComponentController = new EmotesCustomizationComponentController();
+            IEmotesCustomizationComponentView emotesSectionView = emotesCustomizationComponentController.Initialize(
+                dataStore.emotesCustomization,
+                dataStore.emotes,
+                dataStore.exploreV2,
+                dataStore.HUDs);
+
+            emotesSectionView.viewTransform.SetParent(emotesSectionTransform, false);
```

Comment:
i dont think that the controller should have the responsibility of parenting transforms.. could it be done in the prefab instantiation process?

Reply:
This has been solved in the previous change (https://github.com/decentraland/unity-renderer/pull/4933/commits/f176b5442e99e5b1b910728c23f6998e0b19d520). Now this parenting is done in the view creation 👍